app/views/donations/show.html.erb

```diff
@@ -49,7 +49,28 @@
       </div>
     </div>
   </div>
-
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-12">
+        <div class="card">
+          <div class="card-header">
+            <h2 class="card-title">Notes</h2>
+          </div>
+          <div class="card-body p-0">
+              <div class="card">
+                <div class="col-xs-12 col-sm-12" id="donation-notes">
+                  <% if @donation.comment %>
+                    <p><%= simple_format(@donation.comment) %></p>
```

Comment:
```suggestion
                    <p class='px-1'><%= simple_format(@donation.comment) %></p>
```
The donation comment seems a bit unaligned with the card header. I wonder if this px-1 class will add the padding to align it. Let me know how that turns out. If not, I don't think its a big issue.

Reply:
It didn't look as if adding that was affecting anything but I realized I had an extra card div in there and removing made it look quite a bit better, is this ok?

<img width="1198" alt="Screen Shot 2021-03-30 at 8 33 47 PM" src="https://user-images.githubusercontent.com/65036872/113082193-3ded8e80-9197-11eb-89b5-3d9f79406b1e.png">

Reply:
Hmm it looks the same to me as the snapshot you provided in the description?

Reply:
My bad I thought you were talking about the formatting of the card body which looked like a box sitting on a box. After much css turmoil I finally got it figured out (I'm not a css wizard by any means). I renamed it to comment because notes didnt really make sense, had to drop it to below the item table because it was resizing the item table for some reason, and added a stylesheet for donations with a div for donation-comment in order to get the padding right. Sorry that was so much back and forth for something so simple. I pushed up the changes and this is what it looks like now: 
![Screen Shot 2021-03-31 at 1 06 08 PM](https://user-images.githubusercontent.com/65036872/113197960-90c05800-9222-11eb-8d9b-b61793ec8feb.png)